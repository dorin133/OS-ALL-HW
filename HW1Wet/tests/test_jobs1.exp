smash> smash> smash> smash> smash> smash> [1] sleep 100& : 111005 0 secs
smash> smash: sending SIGKILL signal to 1 jobs:
111005: sleep 100&
