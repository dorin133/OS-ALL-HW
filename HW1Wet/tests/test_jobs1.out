smash> smash> smash> smash> smash> smash> [1] sleep 100& : 111009 0 secs
smash> smash: sending SIGKILL signal to 1 jobs:
111009: sleep 100&
